ci/release_notes.md

```diff
@@ -0,0 +1,94 @@
+# New Feature: Kit Providers
+
+Until now, you could only use the kits that were provided by the
+`genesis_community` organization on GitHub, or by creating dev kits yourself.
+This release allows you to specify any GitHub org as a kit provider.
+
+## Creating a new repo with an alternative kit provider:
+
+```
+genesis init --kit-provider        github \
+             --kit-provider-org    myproject \
+             --kit-provider-domain github.mycorp.com \
+             --kit-provider-tls    skip\
+             --kit-provider-name   "Our Custom Kit Provider" \
+             --kit                 my-awesome-kit
+```
+
+There are further options, which can be shown using --help.
+
```

Comment:
Can we document kit providers in detail on genesisproject.io, and link there from here?